spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'ffi'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET "/users/1/posts" ' do
+    before(:example) { get '/users/1/posts' }
+
+    it 'check if the response is successful' do
+      expect(response).to have_http_status(200)
+    end
```

Comment:
- [ ] Let's test `index` and `show` actions for `User` and `Post` controller.
It should look like:
describe 'GET /index' do
  ...
end

  describe 'GET /show' do
     ...
   end